CgfConverter/Renderers/GLTF/GLTF.cs

```diff
@@ -0,0 +1,1513 @@
+﻿using System;
```

Comment:
Woah.  This file is impressive!

Reply:
I think it's better to split each class into different files; will work on it later.

Reply:
Sounds good.  I need to do the same with the Collada Renderer at some point too.